errors_test.go

```diff
@@ -0,0 +1,28 @@
+package duckdb
+
+import (
+	"database/sql"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+var testErrOpenMap = map[string]string{
+	errParseDSN.Error():  ":mem ory:",
+	errOpen.Error():      "?readonly",
+	errSetConfig.Error(): "?threads=NaN",
+}
+
+func TestErrOpen(t *testing.T) {
+
```

Comment:
I would suggest to go with gofumpt's rule no 2:

https://github.com/mvdan/gofumpt?tab=readme-ov-file#added-rules

I always use gofumpt instead of gofmt, as it is stricter and it alwys removes blank lines at the beginning of a function body.

Reply:
Let's make it a formatting rule, then. The new line after the method declaration is a personal preference, i.e., it increases readability for me, but in the end I don't mind. I'll try to add a check/CI formatter for it.

Reply:
Okay, I've looked a bit into `gofumpt` but I only ran it on this branch. I'll add an issue, maybe someone wants to pick it up. I couldn't find a pluggable CI action like for `golangci-lint`, so we'd have to add it to the CI ourselves.